src/components/Registration/DefineRegistrationSummary.vue

```diff
@@ -85,20 +85,18 @@
       </template>
 
       <!-- Business Number -->
-      <template v-if="isTypePartnership">
-        <v-divider class="mx-6" />
+      <v-divider class="mx-6" />
```

Comment:
It doesn't look like any change here, other than changing indenting.
please don't make changes to formatting, for the sake of changing formatting as it detracts from understanding what the PR is attempting to do.

Reply:
Looks like a conditional was removed, so I'm OK with fixing the indentation of the block.

Reply:
This change is obsolete as we want to continue displaying BN only for GPs.
I'm resolving this conversation.